src/kani-compiler/cbmc/src/goto_program/builtin.rs

```diff
@@ -7,6 +7,7 @@ use super::{Expr, Location, Symbol, Type};
 #[derive(Debug, Clone, Copy)]
 pub enum BuiltinFn {
     Abort,
+    Assert,
```

Comment:
All the changes to this file are no longer needed, right? The new `assert` generates a statement, rather than a built-in function call.

Reply:
+1, and shouldn't the opposite be happening in that `CProverAssert` (and `CProverAssume`) should be removed?